cli/generators/component/templates/component.js

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes} from 'react';
+
+const propTypes = {
+};
+
+const defaultProps = {
+};
+
+class <%= name %> extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div>
+                ---Autogenerated <%= name %>---
+            </div>
+        );
+    }
+}
+
+<%= name %>.propTypes = propTypes;
+<%= name %>.defaultProps = defaultProps;
```

Comment:
...water...

Unfortunately I don't know the insights of this yet. I'm discovering it step by step. It work like a class property and I bet is a convention in the React community to put it outside. I can be wrong anyways.